db/migrate/20221129162308_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] This table should refer to `:author`, this will create author_id and a corresponding index and then the foreign_key to the users' table. Please kindly fix that 🙃 😉